### PR TITLE
Removing benchmarks from the main repo.

### PR DESCRIPTION
**Pull request by vjovanov**
Review by @gzm0

### CODE REVIEW

**Comment by gzm0** [comment]
I'm assuming you have put these guys elsewhere. Could you please indicate that in the commit message or somewhere in the README?

**Comment by vjovanov** [comment]
It is added to the README and to the wiki.

File: README.md [hunk 1]
```diff
@@ -121,6 +121,10 @@ It is "safe" to use Scala Records in your project. They cross-compile against al
 
    + If two queries return the same records the results can not be directly combined under a same type. For example, `List(Rec("a" -> 1), Rec("a" -> 2))` will not be usable.
 
+## Perofrmance
```
Comment (gzm0) [inline README.md]: Typo

**Comment by gzm0** [comment]
LGTM otherwise

**Comment by gkossakowski** [comment]
What was the reason to move benchmarks to separate repo?

**Comment by vjovanov** [comment]
They were increasing complexity of the library as they use code generation and external dependencies. 

This way it is easier to evolve the library for us, and to understand the repo for the community.

**Comment by gkossakowski** [comment]
Ok. Thanks!

**Pull request merged**